scripts/config-dhcp.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#copy the configuration information to network-conf.txt
+echo "config-server" > /home/pi/network-conf
+
+#Remove old configuration information if exists from default file
+sudo sed -i '/INTERFACES=/d' /etc/default/isc-dhcp-server 
+
+#Source configuration for dhcp server in the default configuration files
+sudo sh -c "echo 'include "/home/pi/dhcp-server-conf";' >> /etc/dhcp/dhcpd.conf"
```

Comment:
we should do this during post-update.sh (one time). If we do it here, we will end up appending the same line to the bottom of the file many times.